packages/devtools_app/lib/src/shared/console/widgets/evaluate.dart

```diff
@@ -310,14 +311,13 @@ class ExpressionEvalFieldState extends State<ExpressionEvalField>
     serviceManager.consoleService.appendStdio('> $expressionText\n');
     setState(() {
       historyPosition = -1;
-      widget.evalService.evalHistory.pushEvalHistory(expressionText);
+      _evalService.appState.evalHistory.pushEvalHistory(expressionText);
```

Comment:
I would avoid giving the illusion that appState is connected to _evalService when it isn't and just reference the global service directly.